include/linux/slub_def.h

```diff
@@ -7,6 +7,7 @@
  *
  * (C) 2007 SGI, Christoph Lameter
  */
+#include <linux/kasan.h>
```

Comment:
Should we send patches to fix KFENCE unrelated things before RFC?

If we can get this fixed in mainline before would be good, as it saves us a patch in the series. WDYT?